FindYourRecipe/FindYourRecipe.Api/Controllers/CategoryController.cs

```diff
@@ -63,6 +67,8 @@ public async Task<IActionResult> UpdateAsync(int id, CreateOrUpdateCategoryReque
 				return NotFound();
 			}
         }
-	}
+
+     
```

Comment:
For some weird reason I cannot add a comment on line 41 so I' adding it here: the `CreateAsync` method also needs to catch any possible exception and return an error if the case.
As an example, the connection to the database might fail when you try to create a category so the user needs to know that his category was not created.